app/helpers/importHelpers.php

```diff
@@ -1350,14 +1350,14 @@ function caPhpExcelGetDateCellContent($po_sheet, $pn_row_num, $pm_col, $pn_offse
 		if(!is_int($pn_offset)) { $pn_offset = 0; }
 
 		if(!is_numeric($pm_col)) {
-			$pm_col = PHPExcel_Cell::columnIndexFromString($pm_col)-1;
+			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col)-1;
```

Comment:
In PHPSpreadsheet coordinates are 1 based, so the code should read
```suggestion
			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col);
```